File: Learning/OtherLanguages/C/C练习题/Use C/14.c
#include <stdio.h>
#include <math.h>

void main()
{
    int i, n;

    printf("Input a number:");
    scanf("%d", &n);

    printf("\n%d=", n);
    for(i=2; i<n; )
    {

        if(0 == n%i)
        {
            n = n / i;
            printf("%dx", i);
        }
        else
        {
            i++;
        }
    }
    printf("%d\n", n);
}

File: Learning/OtherLanguages/C/C练习题/Use Python/14.py
#因式分解

n = int(input('请输入一个正整数：'))
print('%d = ' % n ,end='')
k = 2
while k <= n:
    if k == n:
        print(k)
        break
    elif n % k == 0:
        print(k,'* ',end='')
        n = n / k
    else:
        k += 1
        
